rust/agama-cli/src/profile.rs

```diff
@@ -45,15 +61,64 @@ fn validate(path: String) -> anyhow::Result<()> {
 fn evaluate(path: String) -> anyhow::Result<()> {
     let evaluator = ProfileEvaluator {};
     evaluator
-        .evaluate(Path::new(&path))
+        .evaluate(Path::new(&path), stdout())
         .context("Could not evaluate the profile".to_string())?;
     Ok(())
 }
 
-pub fn run(subcommand: ProfileCommands) -> anyhow::Result<()> {
+async fn import(url: String, dir: Option<PathBuf>) -> anyhow::Result<()> {
+    let tmpdir = TempDir::new()?; // TODO: create it only if dir is not passed
+    let output_file = if url.ends_with(".sh") {
+        "profile.sh"
+    } else if url.ends_with(".jsonnet") {
+        "profile.jsonnet"
+    } else {
+        "profile.json"
+    };
+    let output_dir = dir.unwrap_or_else(|| tmpdir.into_path());
+    let reader = ProfileReader::new(url.as_str())?;
+    let contents = reader.read()?;
+    let mut output_path = output_dir.join(output_file);
+    let mut output_fd = File::create(output_path.clone())?;
+    output_fd.write_all(contents.as_bytes())?;
+    if output_file.ends_with(".sh") {
+        let err = Command::new("bash")
+            .args([output_path.to_str().context("Wrong path to shell script")?])
+            .exec();
```

Comment:
`output()` would give you `stdout`, `stderr`, etc.

Reply:
it should not be needed as exec keep current fds so identical stdin, stdout and stderr. It basically replace process.